boards/arm/nucleo_f334r8/doc/nucleof334r8.rst

```diff
@@ -0,0 +1,182 @@
+.. _nucleo_f334r8_board:
+
+ST Nucleo F334R8
+################
+
+Overview
+********
+STM32 Nucleo-64 development board with STM32F334R8 MCU, supports Arduino and ST morpho connectivity.
+
+The STM32 Nucleo board provides an affordable and flexible way for users to try out new concepts
+and build prototypes with the STM32 microcontroller, choosing from the various
+combinations of performance, power consumption and features. 
+
+The Arduino™Uno V3 connectivity support and the ST morpho headers allow to expand easily 
```

Comment:
In other RST formatted files I can find examples where TradeMark is used. Is it forbidden? or it is just a convention used for boards' documentation? If so, the line should look like this: 
"The Arduino* Uno V3...", is that right?